thrift.js

```diff
@@ -46,17 +48,32 @@ var ThriftMap = require('./map').ThriftMap;
 var ThriftConst = require('./const').ThriftConst;
 var ThriftTypedef = require('./typedef').ThriftTypedef;
 
+var validThriftIdentifierRE = /^[a-zA-Z0-9_][a-zA-Z0-9_\.]+$/;
```

Comment:
This allows using "42" as an identifier. The first character can only be `[a-zA-Z_]`.